std/debug.zig

```diff
@@ -736,22 +743,6 @@ fn printLineInfo(
             symbol_name,
             compile_unit_name,
         );
-        if (printLineFromFile(out_stream, line_info)) {
```

Comment:
It was actually intentional that disabling "color" also disabled printing source lines. The idea is that the "color" parameter is choosing between 2 modes:
 * machine readable mode, which does not use color and does not print the line from the file or the caret
 * human readable mode, which uses color and prints the line from the file and the caret

Reply:
Have them as two separate options? When the output is not a TTY, use terse stack traces. But if the output is a terminal, but the user dislikes colours, give them an option to disable just that.

Reply:
Pardon my jumping in to add my $0.02, but doesn't this implicit behavior mean there's no way to pipe such debug output to a pager (or to `vim -` as I often do) without the color code noise? It seems like there should be separate explicit switches for this.

Reply:
These are reasonable proposals to make; let's discuss in a separate issue.